From:	Matthew Garrett <mjg@redhat.com>
Subject: [PATCH] x86: Don't use the EFI reboot method by default
Date:	Wed,  6 Jul 2011 16:52:37 -0400

Testing suggests that at least some Lenovos and some Intels will fail
to reboot via EFI, attempting to jump to an unmapped physical address.
In the long run we could handle this by providing a page table with a
1:1 mapping of physical addresses, but for now it's probably just easier
to assume that ACPI or legacy methods will be present and reboot via those.

Signed-off-by: Matthew Garrett <mjg@redhat.com>
---
 arch/x86/platform/efi/efi.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

--- a/arch/x86/platform/efi/efi.c
+++ b/arch/x86/platform/efi/efi.c
@@ -460,9 +460,6 @@ void __init efi_init(void)
 	x86_platform.set_wallclock = efi_set_rtc_mmss;
 #endif
 
-	/* Setup for EFI runtime service */
-	reboot_type = BOOT_EFI;
-
 #if EFI_DEBUG
 	print_efi_memmap();
 #endif
